app/controllers/ems_container_controller.rb

```diff
@@ -13,10 +13,10 @@ class EmsContainerController < ApplicationController
   after_action :set_session_data
 
   OPENSHIFT_ROUTES = {
-    "hawkular"   => %w(hawkular-metrics openshift-infra),
-    "prometheus" => %w(prometheus prometheus)
+    "hawkular"          => %w(hawkular-metrics openshift-infra),
+    "prometheus"        => %w(prometheus prometheus),
```

Comment:
it's just got changed to: prometheus openshift-metrics  :-)

see:
https://github.com/openshift/openshift-ansible/pull/6060